yarn-project/pxe/src/synchronizer/synchronizer.ts

```diff
@@ -17,9 +28,8 @@ import { NoteProcessor } from '../note_processor/index.js';
  * in sync with the blockchain while handling retries and errors gracefully.
  */
 export class Synchronizer {
-  private runningPromise?: Promise<void>;
+  private runningPromise?: RunningPromise;
```

Comment:
unrelated, but quick cleanup of previous work.

Reply:
Glad that has come in useful. I wrote it ages ago and I think it's in a few places.